yarn-project/circuit-types/src/logs/l1_note_payload/encrypt_buffer.ts

```diff
@@ -14,12 +14,14 @@ import { createCipheriv, createDecipheriv } from 'browserify-cipher';
  *
  * @param ecdhPubKey - The ECDH public key represented as a PublicKey object.
  * @param ecdhPrivKey - The ECDH private key represented as a Buffer object.
- * @param grumpkin - The curve to use for curve operations.
  * @returns A Buffer containing the derived AES secret key.
+ * TODO(#5726): This function is called point_to_symmetric_key in Noir. I don't like that name much since point is not
+ * the only input of the function. Unify naming once we have a better name.
  */
-export function deriveAESSecret(ecdhPubKey: PublicKey, ecdhPrivKey: GrumpkinPrivateKey, curve: Grumpkin): Buffer {
+export function deriveAESSecret(ecdhPubKey: PublicKey, ecdhPrivKey: GrumpkinPrivateKey): Buffer {
```

Comment:
The order of the arguments are opposite of the noir code.
Also, the naming make it sounds like it is the privKey matching the pubKey.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/6020/commits/e35028bd7e5e07dafa43c6a7b7c532c98ee60e9b